interactions-transitions-assignment/part-2-map/us-station-interaction.js

```diff
@@ -21,17 +17,139 @@ function createMap(elementId) {
         .append('g')
         .attr('class', 'svg-group')
         .attr('transform', 'translate(' + margins.left + ',' + margins.top + ')');
-
+    let stations;
     // read in data
     d3.json('us-states.json').then(function (geoJSONStates) {
-        d3.csv('NSRDB_StationsMeta.csv').then(function (stations) {
+        d3.csv('NSRDB_StationsMeta.csv', function(d) {
+            return {
+                stationClass: +d.CLASS,
+                abb: d.ST,
+                elevation: +d['NSRDB_ELEV (m)'],
+                loc: [+d.longitude, +d.latitude]
+            };
+        }).then(function (loadedStations) {
+            stations = loadedStations;
             draw(geoJSONStates, stations);
         });
     });
 
     function draw(geoJson, stations) {
```

Comment:
I would name the stations variable in this function something different in order to differentiate it from the other stations variable declared on line 20